pyproject.toml

```diff
@@ -78,9 +78,9 @@ dependencies = [
 [project.optional-dependencies]
 test = [
     "pytest == 7.1.2",
-    "pytest-xdist == 2.1.0",
-    "pytest-timeout == 1.4.2",
-    "pytest-cov == 2.10.1",
+    "pytest-xdist >= 2.1.0",
+    "pytest-timeout >= 1.4.2",
+    "pytest-cov >= 2.10.1",
```

Comment:
Originally I uncapped them to see if they would fix the failing tests. They didn't fix them, but nothing broke because of them so just kept it.